examples/vacationsApprovalAWSLambdas/README.md

```diff
@@ -0,0 +1,77 @@
+# Vacation Approval App example using AWS Lambdas
+
+With Smartmate you have the ability to call custom endpoints in order to manage custom logic outside the Smartmate environment.
+This example explains how to setup a Smartmate App using a AWS Lambda.
```

Comment:
> This example explains how to setup a Smartmate App using a AWS Lambda.

By reading this text, it's like Smartmate will work on AWS Lambda, which is not the case, we can change it to:
`This example explains how to set up a Smartmate App that uses an AWS Lambda function to execute custom logic.`